test/e2e/suite/runtimeconfiguration/runtimeconfiguration.go

```diff
@@ -81,6 +91,10 @@ var _ = framework.CasesDescribe("runtime configuration", func() {
 	})
 
 	It("it should use the runtime configured issuer", func() {
+		//TODO: paulwilljones fix for Ambient
```

Comment:
question: are you intending to do this before merging this PR?

Reply:
this currently fails for ambient as the `istioctl proxy-config` command to retrieve the certificates is not valid for ztunnel, and `istioctl experimental ztunnel-config certificates` doesn't return the required output. We can block the PR on this until there's a workaround or fix once `istioctl` moves out of experimental.